src/main/c/ssl.c

```diff
@@ -719,8 +719,8 @@ TCN_IMPLEMENT_CALL(jint, SSL, initialize)(TCN_STDARGS, jstring engine)
      * low entropy seed.
      */
     SSL_rand_seed(NULL);
-    /* For SSL_get_app_data2() at request time */
-    SSL_init_app_data2_idx();
+    /* For SSL_get_app_data2() and SSL_get_app_data3() at request time */
```

Comment:
Do we comment anywhere what is intended to be stored in either position?  Seems like this case the `data2` and `data3` are intended to be mutually exclusive?